sympy/integrals/tests/test_laplace.py

```diff
@@ -419,42 +424,40 @@ def simp_hyp(expr):
 
     assert ILT( (s * eye(2) - Matrix([[1, 0], [0, 2]])).inv(), s, t) ==\
         Matrix([[exp(t)*Heaviside(t), 0], [0, exp(2*t)*Heaviside(t)]])
-
-
-def test_inverse_laplace_transform_delta():
-    from sympy.functions.special.delta_functions import DiracDelta
-    ILT = inverse_laplace_transform
-    t = symbols('t')
+    # New tests for rules
+    assert ILT(b/(s**2-a**2), s, t) == b*sinh(a*t)*Heaviside(t)/a
+    assert ILT(b*s/(s**2-a**2), s, t) == b*cosh(a*t)*Heaviside(t)
+    assert ILT(b/(s*(s+a)), s, t) == b*(exp(a*t) - 1)*exp(-a*t)*Heaviside(t)/a
+    assert ILT(b*s/(s+a)**2, s, t) == b*(-a*t + 1)*exp(-a*t)*Heaviside(t)
+    assert ILT(c/((s+a)*(s+b)), s, t) ==\
+        c*(exp(a*t) - exp(b*t))*exp(-t*(a + b))*Heaviside(t)/(a - b)
+    assert ILT(c*s/((s+a)*(s+b)), s, t) ==\
+        c*(a*exp(b*t) - b*exp(a*t))*exp(-t*(a + b))*Heaviside(t)/(a - b)
+    assert ILT(c*s/(d**2*(s+a)**2+b**2), s, t) ==\
+        c*(-a*d*sin(b*t/d) + b*cos(b*t/d))*exp(-a*t)*Heaviside(t)/(b*d**2)
+    # Rules for testing different DiracDelta cases
     assert ILT(2, s, t) == 2*DiracDelta(t)
     assert ILT(2*exp(3*s) - 5*exp(-7*s), s, t) == \
-        2*DiracDelta(t + 3) - 5*DiracDelta(t - 7)
+        2*InverseLaplaceTransform(exp(3*s), s, t, None) - 5*DiracDelta(t - 7)
```

Comment:
I guess these are what you mean by changes about delta though.

This should be an example to be documented in release notes.

Still, I'm not sure if it was returned by customized in inverse laplace transform before, or if it is something naturally get from integral. If it is latter, we somewhat have some inconsistency

Reply:
It is even worse,

```
tau = symbols('tau', nonnegative=True)
meijerint_inversion(exp(s), s, tau)
```

gives `0`, but it should not really converge, should it?

Reply:
I have opened issue #24595 for this.